project-charter.md

```diff
@@ -20,25 +20,25 @@ The mission of the Project is to:
+3.  The GB will encourage transparency (e.g., publish public minutes). Committee and Workgroup meetings will be open to the public, and can be conducted electronically, via teleconference, or in person.
 
-4.  The GB has the authority to create Technical Committees that may focus on code or non-code efforts to advance the Mission (such as standards, specifications, and/or architecture).
+4.  The GB has the authority to create Committees that may focus on code or non-code efforts to advance the Mission (such as standards, specifications, and/or architecture). The Committees of the Project are the “Software Committee,” responsible for technical oversight of inbound and outbound coding projects, and the “Metrics Committee,” responsible for the collection of technology-agnostic metrics and standards to be developed by the Project, and the GB shall provide coordination of the interrelationship between the Software and Metrics Committees.
 
-5.  At the launch of the project, the initial Technical Committees of the project are the “Software Committee,” responsible for technical oversight of inbound and outbound coding projects, and the “Metrics Committee,” responsible for the determination of technology-agnostic metrics and standards to be developed  by the Project, and  the GB shall provide coordination of the interrelationship between the Software and Metrics Committees. At inception of the Project, the voting members of the each Technical Committee consist of those individuals identified on the Project website (http://\[domain name\]). Following the inception of the Project, each Technical Committee is expected to implement procedures and methodologies for the selection of voting members of the Technical Committees.
+5.  Workgroups are estabished and maintained by Project contributors to advance Project work. Workgroups focus on specific metric, methodological, ethical, and technical issues associated with open source project health. 
 
-6.  Roles: Technical Committee projects will generally involve Contributors and Maintainers:
+6.  Roles: Committees and Workgroups involve Contributors and Maintainers:
 
-    1.  Contributors: anyone in the community that contributes code, documentation, use cases, standardized metrics, or other technical artifacts to the Project;
+    1.  Contributors: anyone in the community that contributes code, documentation, use cases, standardized metrics, or other community activities related to the Project;
 
-    2. Maintainers: Contributors who have the ability to commit directly to a Project’s main branch, repository, or wiki page and are responsive to contributions or changes from Contributors.
+    2. Maintainers: Contributors who have the ability to commit directly to a Project’s main repository branch and are responsive to contributions or changes from Contributors. Maintainers are listed in the README of each repository. 
```

Comment:
Maybe we can just say "Project's repository"? In fact, what we can control with permissions in GitHub, is that, not specific branches...

Reply:
+1 Good idea!

Reply:
Done